### PR TITLE
Add 'Days' column and cleanups

### PR DESCRIPTION
**Pull request by alfsb**
This PR does a bit of cleanup, before adding the main functionality: a new "Days" column to indicate how many days passed since the last update on `en/`.

The new column:
image

Sending this PR from another repo because the initial fork contains several other alterations.

### CODE REVIEW

**Review by Girgias — APPROVED** [review]
This makes sense to me, do you have commit rights on doc-base?

**Comment by alfsb** [comment]
Yes, I have. I will run this on most active translations before merging.

**Pull request merged**
